packages/e2e/features/ui/components/authenticator/reset-password.feature

```diff
@@ -30,3 +30,34 @@ Feature: Reset Password
     Then I see "Enter your username"
     And I don't see "Enter your phone number"
     And I don't see "Enter your email"
+
+  @angular @react @vue
```

Comment:
Can you repeat this change in [sign-in-force-new-password.feature](https://github.com/aws-amplify/amplify-ui/blob/main/packages/e2e/features/ui/components/authenticator/sign-in-force-new-password.feature)?

Reply:
Sure will add these in sign-in-force-new-password.feature.

Reply:
Added the scripts.